acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/mining/FixedMiningDifficultyAcceptanceTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.tests.acceptance.mining;
+
+import tech.pegasys.pantheon.tests.acceptance.dsl.AcceptanceTestBase;
+import tech.pegasys.pantheon.tests.acceptance.dsl.account.Account;
+import tech.pegasys.pantheon.tests.acceptance.dsl.node.Node;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class FixedMiningDifficultyAcceptanceTest extends AcceptanceTestBase {
```

Comment:
I think this can just be removed now that dev mode is using the fixed difficulty calculator - pretty much every AT is using it and mining transactions so if it ever breaks they'll all grind to a halt because of the increased difficulty.

Reply:
removed